src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java

```diff
@@ -70,7 +70,7 @@ public void setUp() {
     PowerMockito.mockStatic(Analytics.class);
     when(analytics.getApplication()).thenReturn(context);
     integration = new AdobeIntegration(new ValueMap()
-        .putValue("heartbeatTrackingServer", "true"), analytics, Logger.with(NONE),
+        .putValue("heartbeatTrackingServerUrl", "true"), analytics, Logger.with(NONE),
```

Comment:
This is a confusing example, when glancing at this quickly I would have thought this setting was a boolean, where it is a String. 

Not blocking but would prefer to see a url value as an example here